### PR TITLE
Bump preact to v10

### PR DESCRIPTION
**Pull request by zephraph**
## Overview 

Preact X will be released in the near future and it'd be nice to be able to test it on next projects sooner rather than later. 

I haven't bumped the version (not sure how that process works), but I'll be happy to make whatever change I may need to. 

## Changes

- The `preact-compat` package now lives in `preact/compat` so no other packages are needed
- Preact v10 has native support for `createContext` so no `preact-context` package is needed
- Due to the above changes `preact-compat.js` isn't needed

### CODE REVIEW

File: packages/next-preact/readme.md [hunk 1]
```diff
@@ -34,15 +34,12 @@ require('@zeit/next-preact/alias')()
 const { createServer } = require('http')
 const next = require('next')
 
-
 const app = next({ dev: process.env.NODE_ENV !== 'production' })
 const port = process.env.PORT || 3000
 const handle = app.getRequestHandler()
 
-app.prepare()
```
Comment (zephraph) [inline packages/next-preact/readme.md]: These changes are due to prettier editing the file from my editor... I can revert it if you'd like.

**Comment by zephraph** [comment]
@timneutkens what do you think? I'd love to get this out in the wild and some demos set up before PreactX fully launches. It'd be nice to have a PreactX <3 Next 9 post in some of the launch hype!

**Comment by timneutkens** [comment]
I think we should use the opportunity to move this plugin into the monorepo (https://github.com/zeit/next.js). However I think we can do a better preact integration into Next.js

**Comment by zephraph** [comment]
Okay, cool. Is this mergeable until we figure that part out? I'd be happy to help out with porting it closer to next. Would be awesome to have a tighter integration!

**Comment by timneutkens** [comment]
You can copy the directory into `packages` in next and rename to @next/preact

**Pull request closed without merging**